### PR TITLE
added badges

### PR DESCRIPTION
**Pull request by Brujitari**
Issue #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+[![Build](https://github.com/arshadkazmi42/genql/actions/workflows/nodejs.yml/badge.svg)](https://github.com/arshadkazmi42/genql/actions/workflows/nodejs.yml)
+[![NPM Version](https://img.shields.io/npm/v/genql.svg)](https://www.npmjs.com/package/genql)
+[![NPM Downloads](https://img.shields.io/npm/dt/genql.svg)](https://www.npmjs.com/package/genql)
+[![Github Repo Size](https://img.shields.io/github/repo-size/arshadkazmi42/genql.svg)](https://github.com/arshadkazmi42/genql)
+[![LICENSE](https://img.shields.io/npm/l/genql.svg)](https://github.com/arshadkazmi42/genql/blob/master/LICENSE)
+[![Contributors](https://img.shields.io/github/contributors/arshadkazmi42/genql.svg)](https://github.com/arshadkazmi42/genql/graphs/contributors)
+[![Commit](https://img.shields.io/github/last-commit/arshadkazmi42/genql.svg)](https://github.com/arshadkazmi42/genql/commits/master)
+
+
 [![Build Status](https://api.travis-ci.com/arshadkazmi42/genql.svg?branch=master)](https://api.travis-ci.com/arshadkazmi42/genql)
```
Comment (arshadkazmi42) [inline README.md]: Can you remove this line also, i think this is duplicated

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,923 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (arshadkazmi42) [inline yarn.lock]: We don't want this file. Can revert this, and keep only the changes related to README.md

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for working on this.

I have added few comments, can you fix those and post that we can get this merged

File: README.md (hunk 1 above)
Comment (Brujitari) [reply README.md]: you refer to line 12 right?

Comment (arshadkazmi42) [reply README.md]: yes

**Review by arshadkazmi42 — APPROVED** [review]
LGTM :tada:

**Pull request merged**